### ISSUE TITLE
You are using an unsupported command-line flag: --enable-blink-features=ShadowDOMV0. Stability and security will suffer.

### ISSUE THREAD

**Issue by kmrajay237** [operasoftware/operachromiumdriver#83]

**Comment by Hazzer** [comment]
+1 have similar problem

**Comment by bedantaguru** [comment]
+1 same here

**Comment by JeanMarieK** [comment]
+1 have similar issue

**Comment by pbi-qfs** [comment]
On Chrome and MS Edge, the message disappears when the argument `-test-type` is set (which `chromedriver`, `msedgedriver` and also `operadriver` do), but the Opera browser doesn't care...

**Comment by TheCodcher** [comment]
+1 have similar issue

**Comment by rkochar** [comment]
Hello, do you have an update on this? I noticed that `ctrl + l` (enter text in browser search bar) from the keyboard on the browser allows the automation to continue but it feels like that has been "disabled" in later versions. What is a good work around for this in Python? To clarify, `driver.get(url)` gets stuck as shown in the picture at the top.

**Comment by rkrupski** [comment]
Hi, the problem should be fixed in the latest [release](https://github.com/operasoftware/operachromiumdriver/releases/tag/v.91.0.4472.77) for Opera 77.

**Comment by pbi-qfs** [comment]
I can confirm that the info bar disappears on macOS with Opera 77 if you specify `-test-type=webdriver` (which `operadriver` does).

**Comment by rkochar** [comment]
Indeed, the error message has gone but the browser is still on `data:,` as shown at the top when doing `driver.get(url)`. I downloaded the latest zip file (linux64). I initially thought (like others) that solving "--enable-blink-features=ShadowDOMV0" error would get the browser working. Does `driver.get()` work for you in python? An important difference is that I have selenium 3 unlike the original post.

**Comment by pbi-qfs** [comment]
With the Selenium 3 Java binding, we don't see any issue there, but we also clean up the browser cache in the profile before start. Even more, the `data:,` is required to avoid Opera getting stuck at the `shared speed dial` page (#77), or has that also been fixed?

**Comment by rkochar** [comment]
`data:,` being required is fine, I'm trying to point out that if `driver.get(url)` does not work, it's very hard to do something useful. Manually adding in a url works fine but  I would like to automate everything.
